R/drive_file.R

```diff
@@ -7,68 +7,17 @@
 #' @export
 #'
 drive_file <- function(id = NULL, ...) {
-  request <- build_drive_file(id = id, ...)
+  request <- build_request(
+    endpoint = "drive.files.get",
+    token = drive_token(),
+    params = list(...,
```

Comment:
definitely buying it! I'm adding in the repetition & unique endpoint (re the comment 👇 ) and I'll merge! thank you!